content/en/docs/release/release-v330.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "3.3.0 Release Notes"
+keywords: "Kubernetes, KubeSphere, Release Notes"  
+description: "KubeSphere 3.3.0 Release Notes"  
+linkTitle: "3.3.0 Release Notes"  
+weight: 18098
+---
+
+## DevOps
+### Features
+- Support GitOps-based continuous deployment with Argo CDs as the backend, and users can view the status of continuous deployments in real time.
+- Support the continuous deployment allowlist to restrict the target repository and deployment location for continuous deployments.
+- Support the ability to import and manage code repositories.
+- Add built-in CRD-based pipeline templates and support parameter customization.
+- Support the ability to view pipeline events.
+
+## Storage
+### Features
+- Support for tenant-level storage class permission management.
+- Add features such as volume snapshot content and volume snapshot class management.
+- Support automatic restart of deployments and statefulsets after a PVC has been changed.
+- Support automatic expansion of PVs.
+
+## Multi-tenancy and Multi-cluster
+### Features
+- Support for kubeconfig update for member clusters on the KubeSphere console.
+- Support for notifications before a cluster's certificate is about to expire.
+- Support for cluster-level member management.
+
+## Observability
+
+### Features
+- Add container process/thread monitoring metrics.
+- Support node usage monitoring.
+- Support for the ability to import the Grafana template.
```

Comment:
The following content is missing:

- Add support for defining separate data retention periods for container logs, resource events, and audit logs. ([ks-installer#1915](https://github.com/kubesphere/ks-installer/pull/1915), [@wenchajun](https://github.com/wenchajun))